packages/trpc/server/routers/loggedInViewer/outOfOffice.schema.ts

```diff
@@ -16,6 +17,22 @@ export type TOutOfOfficeInputSchema = z.infer<typeof ZOutOfOfficeInputSchema>;
 
 export const ZOutOfOfficeDelete = z.object({
   outOfOfficeUid: z.string(),
+  userId: z.number().nullable().optional(),
 });
 
 export type TOutOfOfficeDelete = z.infer<typeof ZOutOfOfficeDelete>;
+
+export enum OutOfOfficeRecordType {
+  CURRENT = "current",
+  PREVIOUS = "previous",
+}
+
+export const ZOutOfOfficeEntriesListSchema = z.object({
+  limit: z.number().min(1).max(100),
+  cursor: z.number().nullish(),
+  fetchTeamMembersEntries: z.boolean().optional().default(false),
+  searchTerm: z.string().optional(),
+  recordType: z.enum(["current", "previous"]).default(OutOfOfficeRecordType.CURRENT),
```

Comment:
```suggestion
  recordType: z.nativeEnum(OutOfOfficeRecordType).default(OutOfOfficeRecordType.CURRENT),
```

Why not use nativeEnum here?